packages/block-editor/src/components/link-control/index.js

```diff
@@ -404,20 +405,27 @@ function LinkControl( {
 								createSuggestionButtonText
 							}
 							hideLabelFromVision={ ! showTextControl }
+							// Passing the Button component as a suffix prop
+							suffix={
+								showActions ? undefined : (
+									<InputControlSuffixWrapper variant="control">
+										<Button
+											// TODO: Switch to true (40px size) if possible
+											__next40pxDefaultSize={ false }
+											onClick={
+												isDisabled ? noop : handleSubmit
+											}
+											label={ __( 'Submit' ) }
+											icon={ keyboardReturn }
+											className="block-editor-link-control__search-submit"
+											aria-disabled={ isDisabled }
+											size="small"
+										/>
+									</InputControlSuffixWrapper>
+								)
+							}
+							props
 						/>
-						{ ! showActions && (
-							<div className="block-editor-link-control__search-enter">
```

Comment:
If we're removing the `block-editor-link-control__search-enter` classname, we should be able to remove the associated styles as well:

https://github.com/WordPress/gutenberg/blob/cd77dd950b7762ebfa99987ffb0b3ed8f9888d94/packages/block-editor/src/components/link-control/style.scss#L97-L101

Reply:
Yes.